--- a/src/gcc/config/arc/arc-arch.h
+++ b/src/gcc/config/arc/arc-arch.h
@@ -22,6 +22,8 @@ along with GCC; see the file COPYING3.
 #ifndef GCC_ARC_ARCH_H
 #define GCC_ARC_ARCH_H
 
+#include <stddef.h>
+
 #ifndef IN_LIBGCC2
 /* Architecture selection types.  */
 
@@ -116,7 +118,7 @@ typedef struct
 
 } arc_cpu_t;
 
-const arc_arch_t arc_arch_types[] =
+static const arc_arch_t arc_arch_types[] =
   {
     {"none", BASE_ARCH_NONE, 0, 0},
 #define ARC_ARCH(NAME, ARCH, FLAGS, DFLAGS)	\
@@ -126,7 +128,7 @@ const arc_arch_t arc_arch_types[] =
     {NULL, BASE_ARCH_END, 0, 0}
   };
 
-const arc_cpu_t arc_cpu_types[] =
+static const arc_cpu_t arc_cpu_types[] =
   {
     {"none", NULL, PROCESSOR_NONE, 0, HAS_NONE, ARC_TUNE_NONE},
 #define ARC_CPU(NAME, ARCH, FLAGS, EXTRA, TUNE)				\
