--- a/src/libstdc++-v3/python/Makefile.am
+++ b/src/libstdc++-v3/python/Makefile.am
@@ -26,7 +26,7 @@ include $(top_srcdir)/fragment.am
 if ENABLE_PYTHONDIR
 pythondir = $(prefix)/$(python_mod_dir)
 else
-pythondir = $(datadir)/gcc-$(gcc_version)/python
+pythondir = $(datadir)/gcc/python
 endif
 
 if GLIBCXX_BUILD_DEBUG
--- a/src/libstdc++-v3/python/Makefile.in
+++ b/src/libstdc++-v3/python/Makefile.in
@@ -409,7 +409,7 @@ WARN_CXXFLAGS = \
 
 # -I/-D flags to pass when compiling.
 AM_CPPFLAGS = $(GLIBCXX_INCLUDES) $(CPPFLAGS)
-@ENABLE_PYTHONDIR_FALSE@pythondir = $(datadir)/gcc-$(gcc_version)/python
+@ENABLE_PYTHONDIR_FALSE@pythondir = $(datadir)/gcc/python
 @ENABLE_PYTHONDIR_TRUE@pythondir = $(prefix)/$(python_mod_dir)
 @GLIBCXX_BUILD_DEBUG_FALSE@debug_gdb_py = 
 @GLIBCXX_BUILD_DEBUG_TRUE@debug_gdb_py = YES
