--- gcc-13.1.0/src/libstdc++-v3/scripts/run_doxygen	2023-04-26 00:09:43.000000000 -0700
+++ gcc-13.1.0/src/libstdc++-v3/scripts/run_doxygen.new	2023-04-30 14:34:47.513318389 -0700
@@ -130,7 +130,9 @@
 latex_cmd=
 enabled_sections=
 generate_tagfile=
-DATEtext=`date '+%Y-%m-%d'`
+DATE_FMT="+%Y-%m-%d"
+SOURCE_DATE_EPOCH="${SOURCE_DATE_EPOCH:-$(date +%s)}"
+DATEtext=`date -u -d "@$SOURCE_DATE_EPOCH" "$DATE_FMT" 2>/dev/null || date -u -r "$SOURCE_DATE_EPOCH" "$DATE_FMT" 2>/dev/null || date -u "$DATE_FMT"`
 
 # Show how this script is called.
 echo run_doxygen $*
